core/amber/src/main/scala/edu/uci/ics/amber/engine/common/client/ClientActor.scala

```diff
@@ -105,6 +106,8 @@ private[client] class ClientActor extends Actor with AmberLogging {
     case x: WorkflowRecoveryMessage =>
       sender ! Ack
       controller ! x
+    case x: WorkflowRecoveryStatus =>
+      handleControl(x)
```

Comment:
I know you want to bypass the control channel AND use a control handler to handle this message. is there a better way to do so? this message is for the receiver (instead of dp) to handle, can it be an ActorMessage?

Reply:
This is on the client-actor side, and it is only for hooking up with the old recovery status logic. Will refactor it later.